core/src/main/kotlin/de/fraunhofer/iem/spha/core/strategy/XorKPICalculationStrategy.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Fraunhofer IEM. All rights reserved.
+ *
+ * Licensed under the MIT license. See LICENSE file in the project root for details.
+ *
+ * SPDX-License-Identifier: MIT
+ * License-Filename: LICENSE
+ */
+
+package de.fraunhofer.iem.spha.core.strategy
+
+import de.fraunhofer.iem.spha.core.hierarchy.KpiHierarchyEdge
+import de.fraunhofer.iem.spha.model.kpi.KpiStrategyId
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiCalculationResult
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiNode
+
+internal object XorKPICalculationStrategy : BaseKpiCalculationStrategy() {
+
+    override val kpiStrategyId: KpiStrategyId
+        get() = KpiStrategyId.XOR_STRATEGY
+
+    override fun internalCalculateKpi(edges: Collection<KpiHierarchyEdge>): KpiCalculationResult {
+
+        if (edges.size != 2) {
+            return KpiCalculationResult.Error(
+                "XOR calculation strategy called " +
+                    "with ${edges.size} valid elements, which is illegal."
+            )
+        }
+
+        if (
+            (edges.first().to.score == 100) && (edges.last().to.score != 100) ||
+                (edges.first().to.score != 100) && (edges.last().to.score == 100)
```

Comment:
This is preference, but it could be more concise with `!=` instead.
```suggestion
            (edges.first().to.score == 100) != (edges.last().to.score == 100)
```